Core/GDCore/IDE/WholeProjectRefactorer.cpp

```diff
@@ -1628,19 +1628,18 @@ void WholeProjectRefactorer::ObjectOrGroupRemovedInEventsFunction(
 
 void WholeProjectRefactorer::ObjectOrGroupRenamedInEventsBasedObject(
     gd::Project& project,
-    gd::EventsBasedObject& eventsBasedObject,
     gd::ObjectsContainer& globalObjectsContainer,
```

Comment:
Open question: is the global objects container even making sense in an events based object?

Reply:
No, the GUI hold an empty one to fill the parameter (like behaviors do).